### PR TITLE
feat(back): use sqlboiler to generate views

### LINKED ISSUE #23: Back - dig usage of templates for sqlboiler to simplify views

### PR DESCRIPTION
**Pull request by yyewolf**
### What does this PR do?

- use sqlboiler to generate views, simplifies the process
- use comments to store additional tags onto structs

### CODE REVIEW

File: back/templates/custom/00_struct.go.tpl [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	{{- else -}}
+		{{ $colAlias }} *{{ $column.Type -}}
+	{{- end -}}
+
+	{{- /*
+	  handle struct tags
+	  StructTagCasing will be replaced with $.StructTagCases
+	  however we need to keep this backward compatible
+	  $.StructTagCasing will only be used when it's set to "alias"
+    */ -}}
```
Comment (Nounoursdestavernes) [inline back/templates/custom/00_struct.go.tpl:30]: Why comment ? deadcode or for futur pr ?

Comment (yyewolf) [reply back/templates/custom/00_struct.go.tpl:30]: It's taken from SQLBoiler's template (as we are deriving from it), I can remove it if necessary

Comment (Nounoursdestavernes) [reply back/templates/custom/00_struct.go.tpl:30]: Not fan of having commented code. Bad practice in my opinion.

At least must have a clear description that is from the SQLBoiler's template

**Review by NathanCornelie — CHANGES_REQUESTED** [review]
remove comments if not necessary; better to update the documentation but as you want 

I mean it's not a big deal if the comments are temporary

**Comment by yyewolf** [comment]
> remove comments if not necessary; better to update the documentation but as you want
> 
> I mean it's not a big deal if the comments are temporary

I'll update the doc, nice catch 🎣

**Review by NathanCornelie — APPROVED** [review]

**Pull request merged**